openfecwebapp/templates/advanced.html

```diff
@@ -120,6 +133,12 @@ <h1 class="icon-heading__title">{{ title }}</h1>
       <div id="spending" class="option">
         <h2>Spending</h2>
         <div class="option__content">
+          <p>Spending includes all the types of <span class="term" data-term="disbursement">dibsursements</span> candidates and committees make.</p>
+          <p>Search all disbursements to candidates and committees by the recipient, purpose, amount and date.</p>
```

Comment:
Sorry, I should have caught this sooner: this should be "Search all disbursements **from** committees..." right?